### PR TITLE
*: independent the client option

### PR DESCRIPTION
**Pull request by JmPotato**
### What problem does this PR solve?

Issue Number: ref #8690.

### What is changed and how does it work?

```commit-message
Independent the client option to reduce the number of files under the client root directory.
```

### Check List

Tests

- Unit test
- Integration test

### Release note

```release-note
None.
```

### CODE REVIEW

**Review by okJiang — APPROVED** [review]

File: client/opt/option.go [hunk 1]
```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package pd
+package opt
```
Comment (rleungx) [inline client/opt/option.go:15]: Will it have a compatibility issue?

Comment (JmPotato) [reply client/opt/option.go:15]: Yes, when upgrading the downstream components, manual modification of the relevant code is required.

**Review by rleungx — APPROVED** [review]

**Pull request merged**